public_html/index.php

```diff
@@ -33,7 +33,9 @@
 */
 
 require_once ( 'php/Widar.php' ) ;
-$widar = new Widar ( 'widar' ) ;
+$widar = new Widar ( 'widar', function () {
```

Comment:
I guess this now needs to be tweaked to take the plain value rather than a function

Reply:
Done so.